JSDemos/Demos/DateBox/Formatting/description.md

```diff
@@ -1 +1,14 @@
-This demo shows how to&nbsp;use a&nbsp;locale-dependent format (or&nbsp;default format), <a href="/Documentation/ApiReference/Common/Object_Structures/format/#type">built-in format</a>, <a href="/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String">LDML&nbsp;pattern</a>, and a&nbsp;format with literal characters to&nbsp;specify the <a href="/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#displayFormat">displayFormat</a>. Note that when the <a href="/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#useMaskBehavior">useMaskBehavior</a> option is&nbsp;set to&nbsp;<b>true</b>, the <b>DateBox</b> formats the display value and also controls user input.
\ No newline at end of file
+To customize the **DateBox** format, use the **displayFormat** property. This demo illustrates the following use cases of this property:
+- **Locale-dependent format**     
+This is the default formatting behavior dependent on the User's [locale](/Documentation/Guide/Common/Localization/#Localize_Dates_Numbers_and_Currencies).    
+
+- **Built-in predefined format**    
+DevExtreme supports predefined [formats](https://js.devexpress.com/Documentation/ApiReference/Common/Object_Structures/format/#type). Set the **displayFormat** property to the respective format.    
+
+- **LDML pattern**    
+Set the **displayFormat** property to a custom format string using [LDML](/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String) patterns.    
+
+- **Format with literal characters**    
+The **displayFormat** property can contain literal characters combined with **LDML** characters. Wrap the literal characters in quotation marks.
+
+Refer to the [displayFormat](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#displayFormat) description for additional information.
```

Comment:
We forgot to mention `useMaskBehavior`. Let's add information about it as the last paragraph.